CSharp/BatchExplorer/Messages/NodeSchedulingMessage.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.BatchExplorer.Messages
+{
+    public enum SchedulingType
```

Comment:
I suppose it would, though in these cases I like to reduce the number of message classes if possible to help avoid class bloat in the project.